source/js/controllers/search_controller.js

```diff
@@ -46,13 +46,24 @@ app.Controllers.SearchController = Marionette.Controller.extend({
 
     if(cad === 'nearest') {
       this.searchByLocation(this.page);
-    } else if(/^\d{5}(-\d{4})?$/.test(cad)) {
-      this.searchByZipcode(cad, this.page);
     } else {
-      this.searchByName(cad, this.page);
```

Comment:
Could be, but I left it just in case it was decided we really want that somewhere. I'm on board for dumping it too, though.